ui/src/components/EventConfiguration/EventConfigSourceTable.vue

```diff
@@ -161,8 +161,8 @@ const emptyListContent = {
 const columns = computed(() => [
   { id: 'fileName', label: 'Name' },
   { id: 'description', label: 'Description' },
-  { id: 'fileOrder', label: 'File Order' },
```

Comment:
I think here we can:

- remove `fileOrder` column entirely
- move `vendor` column between `filename` and `description`

We will only display file order in the screen where user can reorder event config sources.